connectors/protocol/connectors.py

```diff
@@ -724,7 +724,7 @@ async def sync_done(self, job, cursor=None):
         await self.index.update(doc_id=self.id, doc=doc)
 
     @with_concurrency_control()
-    async def prepare(self, config):
+    async def prepare(self, config, sources):
         """Prepares the connector, given a configuration
         If the connector id and the service type is in the config, we want to
         populate the service type and then sets the default configuration.
```

Comment:
>Function docstring should be updated - it still refers to a single connector

It's still a single connector as you call it like `connector.prepare(...`

But I agree the docstring should be updated. Will do

Reply:
>Function docstring should be updated

It actually doesn't change what the method does. Because `sources` are not in `config` anymore, we need to pass in a new param `sources`.

>it still refers to a single connector

It's still a single connector, we call the method like `connector.prepare(...`